package.json

```diff
@@ -22,6 +22,6 @@
   "license": "MIT",
   "devDependencies": {
     "better-assert": "^1.0.2",
-    "mocha": "^7.1.1"
+    "mocha": "~1.7.4"
```

Comment:
not sure about this one

Reply:
Downgraded to fix existing broken CI; due to mocha v7 not working on Node 6; which is still supported by the module. https://github.com/then/is-promise/pull/15#issuecomment-619403970

Reply:
ah - mocha 3 should still work, it's mocha 4 where it stopped being useful for things that like back compat :-)